docs/dev/developer-guides/transactions/blocks.md

```diff
@@ -0,0 +1,68 @@
+# General
+
+A block in Ethereum is a list of transactions that have been recorded historically. They are collections of transactions that include the hash of the chain's previous block.
+We group transactions into blocks to ensure everyone on the network stays in sync and agrees on the detailed history of transactions. It implies that hundreds of transactions are simultaneously committed, decided upon, and synced.
+
+## Prerequisite
+
+To fully understand this page, we suggest reading [accounts](account.md), [transactions] and (transactions.md).
+
+## How do Ethereum blocks work?
+
+Blocks are strictly ordered (each new block has a connection to its parent block), and transactions within blocks are strictly ordered to keep track of the history of transactions.
+Unless there are exceptional circumstances, all network participants agree with the precise number and history of blocks at any particular time and attempt to combine the pending live transaction requests into the upcoming block.
+
+
+Every node adds the newly created block to the end of its blockchain after it has been assembled by one of the network's validators, and a new validator is chosen to build the following block.
+
+
+## Block numbers and time
+
+On zkSync, when a user sends L2 transfers to the operator, they are not processed immediately.
+Instead, they are included in a group of pending transactions in the available off-chain state. The user receives witnesses to their balance and all pending transactions in the new rollup state when the operator updates the rollup state. The blockchain does not contain these witnesses.
+The user signs and sends an operation labeled `ProcessTransactions` to the operator to have their pending transactions processed.
```

Comment:
Maybe I don't fully understand what you mean by witness for the balance, but I think we don't have such functionality. The same for `ProcessTransactions`, `lastSeenBlockNum(sender)`, we do not have it in ZkSync